datayoga-py/datayoga/blocks/read_redis_stream/block.py

```diff
@@ -21,6 +21,7 @@ def init(self, context: Optional[Context] = None):
             connection.get("password"))
 
         self.stream = self.properties["stream_name"]
+        self.read_once = self.properties.get("read_once", False)
```

Comment:
what does this property mean? saw it before and didn't understand

Reply:
This is used in the integration test, an indication to read from the stream only once and exit instead of the infinite loop.